### PR TITLE
Doc: Add usage of cli under embedding

### LINKED ISSUE #346: use ansys-mechanical console script in user_guide_embedding

### PR DESCRIPTION
**Pull request by dipinknair**
Add usage of `ansys-mechanical` cli for executing a python script

Closes #346

### CODE REVIEW

File: doc/source/user_guide_embedding/index.rst [hunk 1]
```diff
@@ -103,4 +103,21 @@ From a terminal, run this command:
 
     /usr/ansys_inc/v231/aisol/.workbench -DSApplet -AppModeMech -nosplash -notabctrl -script file.py
 
+.. note::
+   ``-nosplash`` and ``-notabctrl`` not needed for versions 23.2 or above
```
Comment (koubaa) [inline doc/source/user_guide_embedding/index.rst:107]: @dipinknair I think we should discuss `ansys-mechanical` before `.workbench`.  `.workbench` can be just a footnote or note, when explaining what `ansys-mechanical` does.

**Review by koubaa — APPROVED** [review]

**Review by sbradrick — APPROVED** [review]

File: doc/source/user_guide_embedding/index.rst (hunk 1 above)
Comment (dipinknair) [reply doc/source/user_guide_embedding/index.rst:107]: @koubaa  let me know if this is the change what you suggested.

|  Current | New  |
|---|---|
|  image | image |

**Review by koubaa — APPROVED** [review]

File: doc/source/user_guide_embedding/index.rst [hunk 2]
```diff
@@ -74,33 +74,49 @@ Mechanical's command line interface. For instance, the consider the following Py
...
 
 .. code::
 
-    "C:/Program Files/ANSYS Inc/v231/aisol/bin/winx64/AnsysWBU.exe -DSApplet -AppModeMech -script file.py"
+    ansys-mechanical -i file.py
 
 .. note::
+
+   Alternately user can use below commands with
+   command prompt and terminal in Windows and Linux systems respectively.
```
Comment (sbradrick) [inline doc/source/user_guide_embedding/index.rst:102]: Alternately user can use the following commands in the command of the Windows platform and the terminal for Linux systems.

Comment (dipinknair) [reply doc/source/user_guide_embedding/index.rst:102]: I think I missed this, Will do this in next PR.

**Pull request merged**